src/store/model-store.js

```diff
@@ -98,7 +98,14 @@ function savePhoto(b64Str, address) {
     fs.writeFileSync(photoPath, bitmap);
     return photoPath;
 }
-
+function checkoutModels(equipmentAddresses){
```

Comment:
Nit: this name is a bit deceiving; it's not actually checking out model, it only decrements the amount in stock